internal/services/exchange.service.go

```diff
@@ -37,7 +42,21 @@ func getNBUData(date string) []models.NBU {
 	return response
 }
 
-func GetNBUExchange() models.ExchangeRatesResponse {
+func isExistItemInArray(arr []string, item string) bool {
```

Comment:
For Golang 1.21 you can use https://pkg.go.dev/slices#Contains

Reply:
Fixed it. Thank you very much for the advice and for showing me a very useful standard library)